SPARQLWrapper/SmartWrapper.py

```diff
@@ -67,15 +68,15 @@ def __init__(self, variable, binding):
         self.datatype = None
         try:
             self.lang = binding["xml:lang"]
-        except:
+        except KeyError:
             # no lang is set
             pass
         try:
             self.datatype = binding["datatype"]
-        except:
+        except KeyError:
```

Comment:
While this makes sense to me, ideally it would go in another PR as it is not needed for type checking to pass. See [[ref](https://github.com/RDFLib/sparqlwrapper/pull/197/files#r810545424)] for discussion.

Reply:
934e334

Reply:
934e334